.github/workflows/maven-central-push.yml

```
@@ -72,6 +72,7 @@ jobs:
         run: |
           cd scs-multiapi-maven-plugin
           mvn deploy -DskipLocalStaging=true -P production --file pom.xml
```

```suggestion
          mvn -B deploy -DskipLocalStaging=true -P production --file pom.xml
```
Using batch mode here as well should make the output easier to read later